artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/components/browse.js

```diff
@@ -491,6 +491,9 @@ var Artemis;
              if (isNaN(timestamp) || typeof timestamp !== "number") {
                 return timestamp;
              }
+             if (timestamp === 0) {
+                return "none";
```

Comment:
thanks for this improvement, what about `N/A`?

Reply:
That would be fine too.
I always avoided N/A because it has 2 meanings (Collins), or 3 relevant meanings (Webster), or even 4 meanings (Wikipedia). But that is more the uncertainty of myself as a non-native speaker :-)
I'll update this PR. Additionally, the only-and-rare occurrence of `n/a` will be replaced by `N/A`.